clients/rkecli/state.go

```diff
@@ -129,7 +128,7 @@ func NewClusterFile(state *cluster.FullState, dirName string, config *Config) (c
 			rkeConfig.Nodes[i].SSHKeyPath = appendSSHPath(rkeConfig.Nodes[i].SSHKeyPath, config.SSHPath)
 		}
 	} else {
-		return "", errors.Wrap(err, "rke SSHPath or SSHKey not configured")
+		return "", errors.Wrap(errors.New("missing SSH configuration"), "rke SSHPath or SSHKey not configured")
```

Comment:
could you change the func definition to be an unnamed error instead of err. Since it isn't being set, its confusing to keep it in.

Reply:
err is being set in the below line https://github.com/rancher/shepherd/blob/main/clients/rkecli/state.go#L142